provider/linear/index.ts

```diff
@@ -49,14 +72,27 @@ const linearIssues: Provider<Settings> = {
             return []
         }
 
-        const linearClient = getLinearClient(settingsInput)
-
-        const issue = await linearClient.issue(issueId)
-        const comments = await issue.comments()
+        const query = `
+            query Issue($id: String!) {
+                issue(id: $id) {
+                    title
+                    description
+                    comments {
+                        nodes {
+                            body
+                        }
+                    }
+                }
+            }
+        `
+        const variables = { id: issueId }
+        const data = await linearApiRequest(query, variables, settingsInput)
```

Comment:
Wrap in try catch as it throws?

Reply:
It's OK to surface these errors on the consumer side. For example, they are [handled](https://github.com/sourcegraph/cody/blob/1e626a602195b421dc25c6da0a68ac4a40269739/vscode/src/editor/utils/editor-context.ts#L295-L297) on the Cody side and printed in the debug channel, and it's helpful to see them there.